bin/debug

```diff
@@ -15,13 +15,25 @@ show_help() {
 }
 
 source "/usr/local/bin/_help"
+source "/usr/local/bin/_common"
 
 ruby_debug_port() {
-  PORT=$(
-    cat .nib | \
-    jq -r ".services[] | select(.name==\"$1\") | .ruby_debug_port"
+  PREFIX=""
+
+  if compose_version2 ; then
+    PREFIX=".services"
+  fi
+
+  DEBUG_SETTING=$(
+    compose_json | \
+      jq "${PREFIX}.$1.environment" | \
+      grep "RUBY_DEBUG_PORT" | \
+      tr ':' '=' | \
+      tr '[,"]' ' '
   )
```

Comment:
The code here is a bit more complicated than I'd like because of the fact that the `environment` key in `docker-compose` can be a hash or array of strings. If it was always a hash then we could have simply used:

```bash
PORT=$(compose_json | jq "${PREFIX}.$1.environment.RUBY_DEBUG_PORT")
```

Instead I'm trying to normalize into something that looks like `RUBY_DEBUG_PORT=3001` and then using a bash [parameter expression](http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_06_02) to extract the port.